diff --git a/omero_marshal/decode/__init__.py b/omero_marshal/decode/__init__.py
index 6689ded..bd54344 100644
--- a/omero_marshal/decode/__init__.py
+++ b/omero_marshal/decode/__init__.py
@@ -9,6 +9,8 @@
 # jason@glencoesoftware.com.
 #
 
+# Needed to avoid import errors when this is the first import
+import omero.all  # noqa
 import omero.model
 import omero.model.enums
 
diff --git a/omero_marshal/encode/__init__.py b/omero_marshal/encode/__init__.py
index 24e2a43..fd56ad4 100644
--- a/omero_marshal/encode/__init__.py
+++ b/omero_marshal/encode/__init__.py
@@ -9,6 +9,9 @@
 # jason@glencoesoftware.com.
 #
 
+
+# Needed to avoid import errors when this is the first import
+import omero.all  # noqa
 from omero import RType
 from omero_model_UnitBase import UnitBase
 from omero.rtypes import unwrap
